### DISCUSSION TITLE
updateListingInventory failing to update

### DISCUSSION THREAD

**Discussion by mikeglas** [etsy/open-api#953] [Q&A]
I can't get the method to update inventory.

I am passing in the headers to

x-api-key [with a valid key]
authorization [with a valid token]
products [{"sku":"11455","offerings":[{"price":4.95,"quantity":3,"is_enabled":1}]}]

**Comment by mikeglas** [comment]
hit enter too soon.

URL: https://openapi.etsy.com/v3/application/listings/{valid listing id}/inventory

using oauth and api-key

In postman - I get "error": "Resource not found"

**Reply by Gareth064** [reply]
Normally in APIs, "Resource Not Found" means the endpoint URI you are calling doesn't exist. But what you posted looks ok to me. Looking at the docs for that endpoint says that error could also mean the listing isnt found too.

Is the listing "Active"?

I know I've seen people have quirks which transpired that you can't do certain actions on inactive listings.

**Reply by mikeglas** [reply]
the listing is active . . . so that's not it.

**Comment by hemesh** [comment]
I had this initially when I was using the POST call. Make sure it's PUT

**Reply by mikeglas** [reply]
that was it - thanks ! Looking back at the docs / sample code - PUT is right there . . .
